extensions/ql-vscode/src/common/commands.ts

```diff
@@ -176,9 +176,7 @@ export type QueryHistoryCommands = {
   "codeQLQueryHistory.sortByCount": () => Promise<void>;
 
   // Commands in the context menu or in the hover menu
-  "codeQLQueryHistory.openQueryTitleMenu": TreeViewTitleMultiSelectionCommandFunction<QueryHistoryInfo>;
```

Comment:
You should be able to remove the `TreeViewTitleMultiSelectionCommandFunction` type now. The only usage should be for `codeQLQueryHistory.itemClicked` and that can now have its type changed to `TreeViewContextMultiSelectionCommandFunction`.

Reply:
Thanks,  I should've checked this.